xdrip-get-entries.sh

```diff
@@ -1285,6 +1300,15 @@ function check_last_glucose_time_smart_sleep()
     if [ $(bc <<< "$sleep_time > 0") -eq 1 -a $(bc <<< "$sleep_time < 240") -eq 1 ]; then
       log "Waiting $sleep_time seconds because glucose records only happen every 5 minutes"
       wait_with_echo $sleep_time
+    elif [ $(bc <<< "$sleep_time < -60") -eq 1 ]; then
```

Comment:
This seems like the appropriate place to determine if backfill tx control message is required. Looks good to me.

Reply:
Ok maybe we can remove my comment below then. I was thinking in the future we might request on the backfilling for gaps rather than just for recent missed entries